examples/kubernetes/manifests/mariadb-secrets.yml

```diff
@@ -0,0 +1,11 @@
+# NOTE: this file should be hidden somehow, or recreated only from the CLI. This
+# has been done this way to illustrate better the example.
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mariadb-secrets
```

Comment:
For the naming of all the objects it would be nice to prefix them with "portus" to avoid conflicts with existing things (and to keep all the portus stuff grouped together visually in kubectl and dashboard)